src/main/resources/init_db.sql

```diff
@@ -1,9 +1,17 @@
-CREATE SCHEMA IF NOT EXISTS `taxi` DEFAULT CHARACTER SET utf8;
-USE `taxi`;
-
+CREATE DATABASE `library_db` /*!40100 DEFAULT CHARACTER SET utf8 */ /*!80016 DEFAULT ENCRYPTION='N' */;
+CREATE TABLE `drivers` (
```

Comment:
Also not sure if it is ok to use library_db in this task, but as I can see your previous task was approved with this naming. Let's also ask mentor's opinion

Reply:
agree, correct!